vcl/extensions.vcl

```diff
@@ -11,14 +11,8 @@
 #
 
 sub hlx_type_pipeline_before {
-    # A non-production XFH header looks like: "repo--user.hlx.page"
-    # If you have an outer CDN, then it looks like "www.mydomain.com, repo--user.hlx.page"
-    # The filter below tests if you have no outer CDN and turns on the development mode
-    # caching (i.e. disables caching in helix-dispatch)
-    if(regsuball(regsuball(req.http.x-forwarded-host, "[^,]*(\.hlx(-[0-9]+)?\.page|\.project\-helix\.page|\.fastlydemo\.net)", ""), "[, ]", "") == "") {
```

Comment:
I'd rather add a `&& false` so that we can re-enable it easily.

Reply:
I can just comment everything :-)